core/src/main/java/org/sonarsource/sonarlint/core/storage/UpdateStorageOnSettingChanged.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.storage;
+
+import java.util.HashMap;
+import org.sonarsource.sonarlint.core.events.ServerEventHandler;
+import org.sonarsource.sonarlint.core.serverapi.push.SettingChangedEvent;
+
+public class UpdateStorageOnSettingChanged implements ServerEventHandler<SettingChangedEvent> {
```

Comment:
Shouldn't you remove that if the SQ team is not implementing it?

Reply:
I will probably yes